### ISSUE TITLE
Combination of policies or customized policies details

### ISSUE THREAD

**Issue by archer29m** [honeynet/droidbot#99]
Hello,

I need to customize the tool in order to be able to bypass login/welcome/ad/etc views. Is there any way so that I can use a combination of policies (e.g., both "none" + "dfs") so that I can feed the inputs manually whenever it's needed? Also in "none" policy, DroidBot doesn't trigger the events properly, right? I don't see any result within this policy. 
But generally, if it needs to be done through customized scripts, how should I fill in the scripts views/states/operations names?

Thank you in advance for your helps.

**Comment by ylimit** [comment]
`droidbot -policy dfs -script xxx.json` would fulfill your need.
The "none" policy means "simply start the app and don't send any input"

**Comment by codeslord** [comment]
The problem is that if you had specified a script then that state will not be taken into consideration for further policy actions. It would be so nice if somebody can combine the power of scripts and policy.

**Comment by archer29m** [comment]
Yes, exactly. That's the main problem I had with "manual scripts" as they do not combine with the main policies very well. 
But in general after a while, I found adding "random" events to be a good alternative for this purpose as it seems to work with the main policies well. But definitely handling the scripts in a way to be properly combined with the actual policy would be very helpful.

Another question I have is that is there any intelligent way to fill the text boxes automatically? For user/pass text boxes, using regexes like "*username" or "*pass" usually works but for other types of text boxes (e.g., search boxes that need to be filled just with a city name or zipcode and not any random text) is there any heuristic way to fill them automatically without necessarily knowing the resource_id for any each app?
